modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -573,16 +575,91 @@ docker volume ls
 docker volume rm {ocis_wopi}_ocis-config {ocis_wopi}_ocis-data
 ----
 
-== Updating
+== Updating and Upgrading
+
+=== Updating
+
+// for testing
+//:version-type: production
 
 ifeval::["{version-type}" == "rolling"]
-Note that this deploymment can, for the time being, only be updated within Infinite Scale rolling releases.
+For rolling releases, the following steps are essential to avoid breaking the setup. This is because rolling releases depend on consecutive updates to be taken.
+
+If new Infinite Scale releases are available, you *must not* skip any version in between the current running and the latest available rolling release for internal update reasons. All versions need to be downloaded and started one time. For more details see the https://owncloud.dev/ocis/release_roadmap/#updating-and-overlap[Updating and Overlap] description in the developer documentation.
+
+* Each upgrade consists of a set of commands:
++
+[source,bash]
+----
+sudo docker compose stop \
+sudo docker compose pull \
+sudo docker compose up -d --remove-orphans
+----
+
+* If there is no release gap, run the command block once.
+
+* For *any* https://owncloud.dev/ocis/release_roadmap/#dates[release gap], you must run the command block from above once and set the respective release in the `OCIS_DOCKER_TAG` _before_ pulling. Use no value for the last release, it defaults to `latest`.
+
+* Check if there are any Infinite Scale config changes. To do so, run a `ocis init --diff` plus apply any patches. The detailed how-to is described in the xref:migration/upgrading-ocis.adoc[Upgrading Infinite Scale] guide for 7.0.0. When done, bring the deployment up.
```

Comment:
Do you want to hard-code 7.0.0 here?
Or remove " for 7.0.0"
Or put some variable that references the current major release number?

Reply:
There will be a new branch for v7 based on master. I have currently fixed that to that version so it is locked. An attribute defining the number dynamically is preferred, but I have not found a solution to cover all variants in master and (upcoming) branches. This is quite complex...